generators/config/templates/capybara.rb

```diff
@@ -39,6 +39,9 @@ module CapybaraHelpers
 
 Capybara.register_driver :selenium do |app|
   params = { browser: Howitzer.selenium_browser.to_s.to_sym }
+  if CapybaraHelpers.chrome_browser? && Howitzer.chrome_headless_mode
+    params[:switches] = %w[--headless --disable-gpu -start-maximized]
```

Comment:
Probably we could use -start-maximized option if Howitzer.maximize_window = true only